tests/circuit_drawer/test_circuit_drawer.py

```diff
@@ -93,6 +93,7 @@ def to_layer(operation_list, num_wires):
     layer = [None] * num_wires
 
     for op in operation_list:
+        # TODO: Currently this function only works if the device's wires are consecutive integers
```

Comment:
When will this be tackled?

Reply:
It is low priority IMO, because we only use this function in tests with consecutive wires at the moment.